io.brackets.Brackets.json

```diff
@@ -0,0 +1,66 @@
+{
+  "app-id": "io.brackets.Brackets",
+  "branch": "stable",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "brackets",
+  "rename-desktop-file": "brackets.desktop",
+  "rename-icon": "brackets",
+  "separate-locales": false,
+  "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=pulseaudio",
+      "--share=network",
+      "--device=dri",
+      "--filesystem=host",
+      "--talk-name=org.freedesktop.Notifications",
+      "--talk-name=org.freedesktop.secrets",
+      "--talk-name=org.gtk.vfs",
+      "--talk-name=org.gtk.vfs.*",
+      "--talk-name=org.gnome.GConf",
+      "--filesystem=host"
+  ],
+  
+  "modules": [
+    {
+      "name": "brackets",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x Brackets.Release.release-1.13-prerelease-3.64-bit.deb",
+        "tar -xf data.tar.xz",
+        "cp -r opt/brackets/* /app",
+
+        "mkdir -p /app/share/icons/hicolor/512x512/apps",
+        "mv usr/share/icons/hicolor/scalable/apps/brackets.svg /app/share/icons/hicolor/512x512/apps",
+	"rm -rf /app/share/icons/HighContrast",
+	
+	"ln -sf /app/brackets /app/bin/brackets",
+	"sed -i 's|\"$HERE/Brackets\"|\"$HERE/Brackets\" --no-sandbox|g' /app/brackets",
+	
+        "desktop-file-edit --set-key=Exec --set-value='brackets %F' --set-key=Icon --set-value=/app/share/icons/hicolor/512x512/apps/io.brackets.Brackets.svg /app/brackets.desktop",
+	"mkdir -p /app/share/applications/",
+	"install -Dm644 /app/brackets.desktop /app/share/applications/brackets.desktop",
+
+        "install -Dm644 io.brackets.Brackets.appdata.xml /app/share/appdata/io.brackets.Brackets.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://github.com/adobe/brackets/releases/download/release-1.13-prerelease-3/Brackets.Release.release-1.13-prerelease-3.64-bit.deb",
```

Comment:
There is also 32 bit deb available. You should also add tarball with source code so it can be installed via Sources extension,

Reply:
Also add `flathub.json` with `only-arches` array to limit on which architectures application will be built.

Reply:
flathub.json was added.

I don't see any reason to support x86 app when x86_64 still available.

Reply:
Because not everyone has x86_64 and we should aim at supporting all 4 architectures we host now.

Reply:
There's 2 ways: someone will add it later or I stop the review right now and will host this app outside from flathub.

Reply:
Added.